src/util/strip.ts

```diff
@@ -23,6 +25,7 @@ export const strip = (
     .replace(regexSpanTagOnlyContainsWhitespaces, '$1') // Replace span tags contain whitespaces
     .replace(regexNbsp, ' ')
     .replace(regexDecimalSpace, ' ') // Some text editors use decimal code for space character
+    .replace(regexExpForEmptyFormattingTags, '') // Replace empty formatting tags with space
```

Comment:
It removes empty formatting tags, not replace them with space.

Reply:
True! Btw do we want to remove empty tags or replace them with spaces?